azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/jobpage.vm

```diff
@@ -49,14 +49,14 @@
       <div class="row">
         <div class="header-title">
           <h1>
-            <a href="${context}/manager?project=${project.name}&flow=${flowid}&job=${jobid}">
+            <a href="${context}/manager?project=${name}&flow=${flowid}&job=${jobid}">
```

Comment:
Given that this and above are the job and flow pages respectively I wonder is `projectName` instead of `name` is more readable. I also understand that using the same variable names for all pages makes the code more predictable and easy to read, so if a different variable name makes sense here can we use the new one in remaining .vm too?

Reply:
I think it makes sense to use projectName instead of name. I will change it.